code/data_structures/src/stack/infix_to_prefix/README.md

```diff
@@ -0,0 +1,31 @@
+# What are Prefix Notations?
```

Comment:
Cite the references.

Reply:
But in other Readme files which were merged, there was no references.??

Reply:
Its a good practice to cite references. @arnavb might be able to guide better on previous readme files.

Reply:
Actually, I am working on OpenGenus IQ article of this topic and I will add the link in another pull request once that article is ready